example/visionos/Podfile

```
@@ -0,0 +1,16 @@
+require_relative '../node_modules/react-native-test-app/visionos/test_app'
+
+workspace 'Example.xcworkspace'
+
+platform :visionos, '1.0'
+
+options = {
+  :fabric_enabled => false,
```

Let's add the `bridgeless_enabled` option as well.
```suggestion
  :bridgeless_enabled => false,
  :fabric_enabled => false,
```